src/lib.rs

```diff
@@ -378,6 +378,13 @@ pub struct Options {
     #[cfg_attr(feature = "clap", clap(long, default_value = "linear"))]
     pub split_strategy: SplitStrategy,
 
+    /// Forces a constant allocation order, instead of a random one. This
+    /// makes it more likely that regalloc will pick the same register for
+    /// for different non-interferring virtual registers. It may increase
+    /// output code quality at the cost of compile time.
+    #[cfg_attr(feature = "clap", clap(long, default_value = "false"))]
+    pub const_allocation_order: bool,
```

Comment:
I think this is actually clearer if it's called `randomize_allocation_order` and default to `true`.